src/xhr.js

```diff
@@ -387,6 +387,15 @@ class FetchResponse {
   }
 
   /**
+   * Drains the response and returns a promise that resolves with the response
+   * text.
+   * @returns {!Promise.<string>}
+   */
+  text() {
+    return this.drainText_();
+  }
```

Comment:
if we need `drainText_` exposed, i think we should just make it public instead of this re-aliasing.

/cc @cramforce